templates/partials/_snap-details-video-layout.html

```diff
@@ -17,3 +17,18 @@
     </div>
   {% endif %}
 </div>
+
+<script>
+  document.addEventListener("DOMContentLoaded", function() {
+    const youtubeVideoFrame = document.getElementById("ytplayer");
+    const vimeoplayerFrame = document.getElementById("vimeoplayer");
```

Comment:
The videos are embedded in a loop. Therefore, I assume there could be multiple with the same `id`?

Reply:
There can only be one video. I've refactored the python to reflect this.